src/java.base/linux/classes/jdk/internal/platform/CgroupUtil.java

```diff
@@ -45,6 +46,8 @@
         } catch (PrivilegedActionException e) {
             unwrapIOExceptionAndRethrow(e);
             throw new InternalError(e.getCause());
+        } catch (UncheckedIOException e) {
+            throw e.getCause();
```

Comment:
There are other reader methods in the same file, should this block be added to them as well?

Reply:
This change takes care of the case when reading of files /proc/self/cgroup or /proc/self/mountinfo might fail due to some reason that can cause the initialization of cgroupv1 subsystem (initSubSystem) to fail. Looking at the usages of other reader methods, I think it makes sense to catch UncheckedIOException there as well.

Reply:
Updated the code changes, and added the catch blocks for the rest of the reader methods and also for the cgroupv2 subsystem.